### PR TITLE
Fixed virtctl console / vnc for namespaces != kubevirt

### PR DESCRIPTION
**Pull request by slintes**
**What this PR does / why we need it**:
Get current namespace dynamically for virtctl console / vnc

**Which issue(s) this PR fixes** :
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1745322

**Special notes for your reviewer**:
Had to move `GetNamespace()` into client-go to prevent duplicate code. 

Since we currently don't use random namespaces in CI, I verified manually that this works now:
```
$ k -n kubevirtx get vmi,pod
NAME                                            AGE   PHASE     IP            NODENAME
virtualmachineinstance.kubevirt.io/vmi-fedora   3m    Running   10.244.0.24   node01
NAME                                   READY   STATUS    RESTARTS   AGE
pod/virt-api-869b56cd-8vbnw            1/1     Running   0          30s
pod/virt-api-869b56cd-r26jg            1/1     Running   0          60s
pod/virt-controller-5746b88598-55fh8   1/1     Running   0          4m14s
pod/virt-controller-5746b88598-5zw4q   1/1     Running   0          4m14s
pod/virt-handler-59pbh                 1/1     Running   0          4m14s
pod/virt-launcher-vmi-fedora-qhhxt     2/2     Running   0          3m33s
pod/virt-operator-74bcfcf78f-547bc     1/1     Running   0          28m
pod/virt-operator-74bcfcf78f-tl7gj     1/1     Running   0          28m

$ ./_out/cmd/virtctl/virtctl -n kubevirtx console vmi-fedora
Successfully connected to vmi-fedora console. The escape sequence is ^]
Fedora 29 (Cloud Edition)
Kernel 4.18.16-300.fc29.x86_64 on an x86_64 (ttyS0)
vmi-fedora login:
```

**Release note**:
```release-note
Fixed virtctl console / vnc for namespaces != kubevirt
```

### CODE REVIEW

File: staging/src/kubevirt.io/client-go/kubecli/handler.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 
 	virtv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/util"
```
Comment (dhiller) [inline staging/src/kubevirt.io/client-go/kubecli/handler.go]: maybe alias this with `clientUtil` like in the other places?

Comment (mfranczy) [reply staging/src/kubevirt.io/client-go/kubecli/handler.go]: but in the other places, we had imported `util` pkg already, I think it's fine.

Comment (slintes) [reply staging/src/kubevirt.io/client-go/kubecli/handler.go]: and at the other places we are not in client-go, here we are, so it is clear which util package this is

**Pull request merged**